### DISCUSSION TITLE
Octo mini

### DISCUSSION THREAD

**Discussion by SimonJones79** [BottlecapDave/HomeAssistant-OctopusEnergy#537] [General]
ive got the octo mini reporting in home assistant i can see the "current demand" sensor, but dont seem to have a current accumulative consumption sensor, has anyone got this working?

**Comment by BottlecapDave** [comment]
Hello. If you can see the demand sensor, the other sensors should also be presen, as they are registered at the same time. Do you have any errors in your logs?

**Comment by SimonJones79** [comment]
Hi Dave thanks for getting back to me,

I have a HA restart set each night, I seem to get these errors after the restart.

Simon

This error originated from a custom integration.

Logger: custom_components.octopus_energy.api_client
Source: custom_components/octopus_energy/api_client.py:432
Integration: Octopus Energy (documentation, issues)
First occurred: 00:00:54 (12 occurrences)
Last logged: 07:00:38

Octopus Energy server error (https://api.octopus.energy/v1/electricity-meter-points/1300061099258/meters/21P9003319/consumption?period_from=2023-11-19T00:00:00Z&period_to=2023-11-20T00:00:00Z): 502; <html> <head><title>502 Bad Gateway</title></head> <body> <center><h1>502 Bad Gateway</h1></center> <hr><center>nginx</center> </body> </html>
Octopus Energy server error (https://api.octopus.energy/v1/electricity-meter-points/1300009202176/meters/21P9003319/consumption?period_from=2023-11-19T00:00:00Z&period_to=2023-11-20T00:00:00Z): 502; <html> <head><title>502 Bad Gateway</title></head> <body> <center><h1>502 Bad Gateway</h1></center> <hr><center>nginx</center> </body> </html>

This error originated from a custom integration.

Logger: custom_components.octopus_energy.api_client
Source: custom_components/octopus_energy/api_client.py:421
Integration: Octopus Energy (documentation, issues)
First occurred: 01:00:42 (22 occurrences)
Last logged: 06:00:47

Octopus Energy server error (https://api.octopus.energy/v1/products/INTELLI-VAR-22-10-14/electricity-tariffs/E-1R-INTELLI-VAR-22-10-14-D/standard-unit-rates?period_from=2023-11-20T00:00:00Z&period_to=2023-11-22T00:00:00Z): 502; <html> <head><title>502 Bad Gateway</title></head> <body> <center><h1>502 Bad Gateway</h1></center> <hr><center>nginx</center> </body> </html>
Failed to send request (https://octopus.energy/api/v1/tracker/E-1R-INTELLI-VAR-22-10-14-D/daily/past/1/0): 400; {"non_field_errors": ["Invalid tariff type."]}
Octopus Energy server error (https://api.octopus.energy/v1/products/OUTGOING-FIX-12M-19-05-13/electricity-tariffs/E-1R-OUTGOING-FIX-12M-19-05-13-D/standard-unit-rates?period_from=2023-11-20T00:00:00Z&period_to=2023-11-22T00:00:00Z): 502; <html> <head><title>502 Bad Gateway</title></head> <body> <center><h1>502 Bad Gateway</h1></center> <hr><center>nginx</center> </body> </html>
Failed to send request (https://octopus.energy/api/v1/tracker/E-1R-OUTGOING-FIX-12M-19-05-13-D/daily/past/1/0): 400; {"non_field_errors": ["Invalid tariff type."]}
________________________________

**Comment by SimonJones79** [comment]
Hi Dave, I've solved it. I had missed an update to HA core, which meant I couldn't see the latest version of your octo plugin. Once I updated HA core, I was presented with the latest version, once updated i now have all the sensors. Thanks for looking at this for me.

**Reply by BottlecapDave** [reply]
Yup that'll do it, I would have suggested this based on the logs. The only other error in the logs is a server error from OE, which is out of my control. Glad that an update had fixed your issue.
